pkg/api/handlers/compat/containers.go

```diff
@@ -403,6 +403,32 @@ func LibpodToContainerJSON(l *libpod.Container, sz bool) (*types.ContainerJSON,
 		state.Status = define.ContainerStateCreated.String()
 	}
 
+	state.Health.Status = inspect.State.Healthcheck.Status
+
+	state.Health.FailingStreak = inspect.State.Healthcheck.FailingStreak
+
+	//item := define.HealthCheckLog{}
+
+	var item (*types.HealthcheckResult)
+
+	log := inspect.State.Healthcheck.Log
+
+	for ind := range log {
```

Comment:
Generally we don't bother with index variables in loops - `for _, item := range log {` will get you the actual healthcheck structure as `item`, no need to index into the array every time

Reply:
Thanks, I'll change to that. I was just doing this loop to convert between `string `and `time.Time` since simply assigning one Log entry to the other wouldn't work because of the incompatible libpod/docker types.